src/globals.h

```diff
@@ -2000,22 +1994,23 @@ extern RangeArrI<int, 1, numCharacters, 0> GFXPlayerWidth;
 extern int PlayerCharacter;
 //Public PlayerCharacter2 As Integer
 extern int PlayerCharacter2;
+
+// replaced with cursor controls
 //Public MenuMouseX As Double
-extern double MenuMouseX;
+// extern double MenuMouseX;
 //Public MenuMouseY As Double
-extern double MenuMouseY;
+// extern double MenuMouseY;
 //! mouse wheel delta
-extern Sint32 MenuWheelDelta;
+// extern Sint32 MenuWheelDelta;
 //! mouse wheel event
-extern bool MenuWheelMoved;
+// extern bool MenuWheelMoved;
 //Public MenuMouseDown As Boolean
-extern bool MenuMouseDown;
+// extern bool MenuMouseDown;
 //Public MenuMouseBack As Boolean
-extern bool MenuMouseBack;
+// extern bool MenuMouseBack;
+
 //Public MenuMouseRelease As Boolean
 extern bool MenuMouseRelease;
-//Public MenuMouseMove As Boolean
-extern bool MenuMouseMove;
```

Comment:
Can you keep all these old variables being commented with giving a comment why they got nuked? They should be a hit for others what they should do at their side to fix the thing. Right now I tried to locally merge abstract controls branch and memory emulator branch into one temp state to try run, and I got errors because some of these global variables were assigned at my memory emulator by address. So, that means, I should put lambdas here to make a trick over.

Reply:
Sure, I can do that.